vision/unit_tests/test_client.py

```diff
@@ -116,6 +116,24 @@ def test_face_detection_from_content(self):
                          image_request['image']['content'])
         self.assertEqual(5, image_request['features'][0]['maxResults'])
 
+    def test_face_detection_from_content_no_results(self):
+        RETURNED = {
+            'responses': [{}]
+        }
+        credentials = _Credentials()
+        client = self._make_one(project=PROJECT, credentials=credentials)
+        client._connection = _Connection(RETURNED)
+
+        image = client.image(content=IMAGE_CONTENT)
+        faces = image.detect_faces(limit=5)
+        self.assertEqual(faces, [])
+        self.assertEqual(len(faces), 0)
+        image_request = client._connection._requested[0]['data']['requests'][0]
```

Comment:
> I'm not entirely sure what you mean here?

- We don't know what's in `client._connection._requested` (i.e. length check)
- We don't know what else is in `client._connection._requested[0]`
- We don't know what else is in `client._connection._requested[0]['requests']`